### ISSUE TITLE
In Sidebar File explorer, keyboard navigation focus is not visible.

### ISSUE THREAD

**Issue by talenelat-elin** [kepano/obsidian-minimal#405]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

5.3.2

### Describe the bug

After using the "Reveal active file in navigation" command from the command palette, the the file or folder focused in the tree upon using the arrow keys to move around, is not made distinct. 

A quick comparision of how it looks in the default theme vs how it looks in Minimal:

https://user-images.githubusercontent.com/71895835/188313416-41ba2498-769c-4731-bbc5-892b9ce2d950.mp4

### Debug info

SYSTEM INFO:
	Obsidian version: v0.15.9
	Installer version: v0.15.6
	Operating system: Windows 10 Enterprise 10.0.19044
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: Minimal
	Snippets enabled: 0
	Restricted mode: on

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please first try updating your theme to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.

**Comment by talenelat-elin** [comment]
Couldn't find a duplicate issue upon searching, but did find #353 when I narrowed down the lines of CSS that were causing the issues. Closing it for being a duplicate.
